workflows/metatdenovo.nf

```diff
@@ -156,13 +157,51 @@ workflow METATDENOVO {
     INPUT_CHECK (
         ch_input
     )
+    .reads
+    .map {
+        meta, fastq ->
+            new_id = meta.id - ~/_T\d+/
+            [ meta + [id: new_id], fastq ]
+    }
+    .groupTuple()
+    .branch {
+        meta, fastq ->
+            single  : fastq.size() == 1
+                return [ meta, fastq.flatten() ]
+            multiple: fastq.size() > 1
+                return [ meta, fastq.flatten() ]
+    }
+    .set { ch_fastq }
```

Comment:
Shouldn't this be indented one level more?

Reply:
I followed the rnaseq indentation here. I am also not sure